### ISSUE TITLE
It reads astrik for requered field which failed in accessibility certificating

### ISSUE THREAD

**Issue by jayleems2018** [OfficeDev/office-ui-fabric-react#6595]
### Please provide a reproduction of the bug in a codepen:
 

Same code:
this.state && this.state.currentProductId && <div>
                    {this.props.supportTopics.length ? <Field
                        name="topic"
                        id="topicOption"
                        component={RenderDropdownField(this.getTopicOptions())}
                        label={Translate('DAC_TOPIC')}
                        required={true}
                        onChange={this.handleTopicChange}
                        validate={[Validators.required]}
                    />
                        : <Spinner />}
               </div>,

### Bug Report 
 It shows and reads asterisk.  Expect: it should read "mandatory".  Further, need to support localization.
 
- __Package version(s)__: (fill this out) 
- __Browser and OS versions__: (fill this out if relevant)
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No)
 
Requested priority: (Blocking, High, Normal, Low) 
 
Products/sites affected: (if applicable)

**Comment by Vitalius1** [comment]
@jayleems2018 
Can you please also provide some screenshots of the issue manifestation. Also, what component are you using. From the code snippet it doesn't look clear. Is it Dropdown?
